core/platform/email/gae_email_services.py

```diff
@@ -34,6 +34,27 @@ def get_incoming_email_address(reply_to_id):
     return 'reply+%s@%s' % (reply_to_id, feconf.INCOMING_EMAILS_DOMAIN_NAME)
 
 
+def is_sender_email_valid(sender_email):
```

Comment:
Prefix with underscore to mark it as module-private. For instructions on testing private functions, see point 9 [here](https://github.com/oppia/oppia/wiki/Writing-backend-tests#guidelines-for-writing-good-tests).

Reply:
Done! Thanks!